tips/concatenation-vs-coercionmd

```diff
@@ -0,0 +1,11 @@
+---
+title: "Concatenation vs Coercion"
+language: "javascript"
+twitter: "@Jkkr1996"
+---
+
+const x = "11" + 1;
+const y = "11" - 1;
+
+console.log(x); // "111"
+console.log(y); // 10
```

Comment:
Do you mind adding some comments explaining what's happening? 🙂

Also, you're missing a `.` in the file name.

Reply:
Sure @telmogoncalves, will add the comments and update the PR